chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/PendingChangeRequestEmailJob.java

```diff
@@ -0,0 +1,225 @@
+package gov.healthit.chpl.scheduler.job;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import javax.mail.MessagingException;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.web.context.support.SpringBeanAutowiringSupport;
+
+import gov.healthit.chpl.changerequest.dao.ChangeRequestDAO;
+import gov.healthit.chpl.changerequest.domain.ChangeRequest;
+import gov.healthit.chpl.dao.CertificationBodyDAO;
+import gov.healthit.chpl.domain.CertificationBody;
+import gov.healthit.chpl.dto.CertificationBodyDTO;
+import gov.healthit.chpl.exception.EntityRetrievalException;
+import gov.healthit.chpl.util.EmailBuilder;
+import gov.healthit.chpl.util.Util;
+
+/**
+ * The PendingChangeRequestEmailJob implements a Quartz job and is available to ROLE_ADMIN and ROLE_ONC. When
+ * invoked it emails configured individuals with the Change Requests that are in a pending state.
+ */
+public class PendingChangeRequestEmailJob extends QuartzJob {
+    private static final Logger LOGGER = LogManager.getLogger("pendingChangeRequestEmailJobLogger");
+    private Properties props;
+
+    @Autowired
+    private ChangeRequestDAO changeRequestDAO;
+
+    @Autowired
+    private CertificationBodyDAO certificationBodyDAO;
+
+    @Autowired
+    private Environment env;
```

Comment:
Could these be moved from field injection to Spring's preferred constructor injection?

Reply:
I'm actually not sure if they can. These are the jobs that are run via Quartz, and they all have weird environment things happening for them. I'll check though

Reply:
Tried this @Plow74 , but the way the Quartz Jobs are created doesn't support the Autowired constructor thing for injection